setup.py

```diff
@@ -42,7 +48,28 @@
 printer = print  # noqa: T001
 
 
+class SetupClean(clean):
+
```

Comment:
PEP8 doesn't make any claims as to whether a blank line is needed here or not.  We generally follow the Google convention - "Don't make the first line of a scope empty".  Not a biggie, there are variations.